recipes/skorch/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "skorch" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f5d912119283de3d96a414cf8523ad6d86de1d8dc6de09379dade9b11b171fd4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - numpy >=1.13.3
+    - pyyaml >=3.12
+    - scikit-learn >=0.19.1
+    - scipy >=0.19.0
+    - tabulate >=0.7.7
+    - tqdm >=4.14.0
+
+test:
+  imports:
+    - skorch
+about:
+  home: https://github.com/dnouri/skorch
+  license: BSD-3
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
Being added to the `MANIFEST.in` with PR ( https://github.com/dnouri/skorch/pull/327 ).

Reply:
Yup, my PR! But it will be too late for `0.3.0`. With the next release I can simply remove the license file copy from the repo.